docs/bestpractices.md

```diff
@@ -103,4 +103,125 @@ If the changes are required to be applied immedietly, the previous inventory job
+
+
+In case agents send traps to the correct address, but there is still no data in the `netops` index, there might some
+issues with credentials. These errors can be seen in logs if the `snmp-splunk-connect-for-snmp-trap` pod. Logs can be 
+accessed either in Splunk in case of using [sck-otel](gettingstarted/sck-installation.md) or directly in kubernetes 
+using the terminal.
+
+#### Accessing logs via Splunk
+If [sck-otel](gettingstarted/sck-installation.md) is installed, run the following search query in Splunk 
+`index=em_logs sourcetype="kube:container:splunk-connect-for-snmp-traps" error`
+
+#### Accessing logs in kubernetes
+to access logs directly in kubernetes, first run `microk8s kubectl -n sc4snmp get pods`. This will output all pods:
+```
+NAME                                                          READY   STATUS    RESTARTS   AGE
+snmp-splunk-connect-for-snmp-worker-trap-99f49c557-j9jwx      1/1     Running   0          29m
+snmp-splunk-connect-for-snmp-trap-56f75f9754-kmlgb            1/1     Running   0          29m
+snmp-splunk-connect-for-snmp-scheduler-7bb8c79855-rgjkj       1/1     Running   0          29m
+snmp-mibserver-784bd599fd-6xzfj                               1/1     Running   0          29m
+snmp-splunk-connect-for-snmp-worker-poller-78b46d668f-59mv4   1/1     Running   0          29m
+snmp-splunk-connect-for-snmp-worker-sender-6f8496bfbf-cvt9l   1/1     Running   0          29m
+snmp-mongodb-7579dc7867-mlnst                                 2/2     Running   0          29m
+snmp-redis-master-0                                           1/1     Running   0          29m
+```
+
+Now get logs of the `snmp-splunk-connect-for-snmp-trap`:
+```
+microk8s kubectl -n sc4snmp logs pod/snmp-splunk-connect-for-snmp-trap-56f75f9754-kmlgb
+```
+
+#### Error messages for different authentication problems
+
+
+##### Wrong community string
+In case of using community string for authentication purposes, the following error should be expected if the arriving trap 
+has a community string not configured in SC4SNMP:
+```
+2024-02-06 15:42:14,885 ERROR Security Model failure for device ('18.226.181.199', 42514): Unknown SNMP community name encountered
+```
+
+If this error occurs, check if the appropriate community is defined under `traps.communities` in `values.yaml`. Example
+`public` community configuration:
+```yaml
+traps:
+  communities:
+    public:
+      communityIndex:
+      contextEngineId:
+      contextName:
+      tag:
+      securityName:
+```
+
+##### Wrong security name or snmp engine id
+
+While sending SNMP v3 traps in case of wrong username or engine id configuration, the following error should be expected: 
+```
+2024-02-06 15:42:14,091 ERROR Security Model failure for device ('18.226.181.199', 46066): Unknown SNMP security name encountered
+```
+
+If this error occurs, make sure that the kubernetes secret with the correct username has been created ([SNMPv3 configuration](configuration/snmpv3-configuration.md)).
```

Comment:
make sure -> verify?
I think this phrase is in every other section, so let's change there too